src/endpoints/pcm.js

```diff
@@ -0,0 +1,13 @@
+import CRUDExtend from '../extends/crud'
+
+class PCMEndpoint extends CRUDExtend {
+  constructor(endpoint) {
+    const config = { ...endpoint } // Need to clone config so it is only updated in PCM
+    config.version = 'experimental'
```

Comment:
We are hardcoding the version in here, as its a beta, and the PCM API is still in development, I feel this is okay for now, however we may need to update this at a later date to be more acceptable.